### PR TITLE
1425: Add Glossary sorting

### LINKED ISSUE #1425: Glossary: Add sorting to the table
Add tablesorter to the glossary table to quickly sort by any column.

### PR DESCRIPTION
**Pull request by pedro-mendonca**
Add sorting to columns Item and Translation.

- [x] Add tablesorter to Glossary view
- [x] Set sorteable columns with properties set within ´<td>´
- [x] Set rows with class `editor` as child rows to keep below the parent on sorting
- [x] Update CSS to only show the arrow sort icon and cursor pointer on sorteable column headers

Fixes #1425

### CODE REVIEW

**Comment by pedro-mendonca** [comment]
@ocean90 initially I've added the script to the bottom of the glossary-view.php as in the [project.php](https://github.com/GlotPress/GlotPress/blob/develop/gp-templates/project.php#L236), but as we have a glossary.js, thought it would be better to add it here.
Let me know what do you think is best.

I left the minifcation for after your review.

**Comment by pedro-mendonca** [comment]
There is still an issue on clicking the details, the row is not in the correct order.

Another thing, might be better to add the 'sorter' ID on the `<td>` to avoid future mapping issues, as some columns are optional, depending on the editing permissions of the user.

**Comment by pedro-mendonca** [comment]
On https://github.com/GlotPress/GlotPress/pull/1426/commits/e78b9a7bee704f6eb7aee43862b1f2602f96a3eb the editor rows are always after the main row, fixing the sorting issue.

File: assets/js/glossary.js [hunk 1]
```diff
@@ -13,6 +13,7 @@ $gp.glossary = (
 				}
 				$gp.glossary.table = table;
 				$gp.glossary.install_hooks();
+				$gp.glossary.tablesorter();
```
Comment (ocean90) [inline assets/js/glossary.js]: Shouldn't we make sorting also available to users you cannot edit entries? I'd move it right below `$gp.init();`.

File: gp-templates/glossary-view.php [hunk 2]
```diff
@@ -61,12 +61,12 @@
...
-			<th class="gp-column-part-of-speech"><?php _ex( 'Part of speech', 'glossary entry', 'glotpress' ); ?></th>
-			<th class="gp-column-translation"><?php _ex( 'Translation', 'glossary entry', 'glotpress' ); ?></th>
-			<th class="gp-column-comments"><?php _ex( 'Comments', 'glossary entry', 'glotpress' ); ?></th>
+			<th class="gp-column-item" data-sorter="text"><?php _ex( 'Item', 'glossary entry', 'glotpress' ); ?></th>
+			<th class="gp-column-part-of-speech" data-sorter=false><?php _ex( 'Part of speech', 'glossary entry', 'glotpress' ); ?></th>
+			<th class="gp-column-translation" data-sorter="text"><?php _ex( 'Translation', 'glossary entry', 'glotpress' ); ?></th>
+			<th class="gp-column-comments" data-sorter=false><?php _ex( 'Comments', 'glossary entry', 'glotpress' ); ?></th>
 			<?php if ( $can_edit ) : ?>
-				<th class="gp-column-actions">&mdash;</th>
+				<th class="gp-column-actions" data-sorter=false>&mdash;</th>
```
Comment (ocean90) [inline gp-templates/glossary-view.php]:
```suggestion
				<th class="gp-column-actions" data-sorter="false">&mdash;</th>
```

File: gp-templates/glossary-view.php [hunk 3]
```diff
@@ -61,12 +61,12 @@
...
 	<thead>
 		<tr>
-			<th class="gp-column-item"><?php _ex( 'Item', 'glossary entry', 'glotpress' ); ?></th>
-			<th class="gp-column-part-of-speech"><?php _ex( 'Part of speech', 'glossary entry', 'glotpress' ); ?></th>
-			<th class="gp-column-translation"><?php _ex( 'Translation', 'glossary entry', 'glotpress' ); ?></th>
-			<th class="gp-column-comments"><?php _ex( 'Comments', 'glossary entry', 'glotpress' ); ?></th>
+			<th class="gp-column-item" data-sorter="text"><?php _ex( 'Item', 'glossary entry', 'glotpress' ); ?></th>
+			<th class="gp-column-part-of-speech" data-sorter=false><?php _ex( 'Part of speech', 'glossary entry', 'glotpress' ); ?></th>
+			<th class="gp-column-translation" data-sorter="text"><?php _ex( 'Translation', 'glossary entry', 'glotpress' ); ?></th>
+			<th class="gp-column-comments" data-sorter=false><?php _ex( 'Comments', 'glossary entry', 'glotpress' ); ?></th>
```
Comment (ocean90) [inline gp-templates/glossary-view.php]:
```suggestion
			<th class="gp-column-comments" data-sorter="false"><?php _ex( 'Comments', 'glossary entry', 'glotpress' ); ?></th>
```

File: gp-templates/glossary-view.php [hunk 4]
```diff
@@ -61,12 +61,12 @@
 <table class="gp-table glossary" id="glossary">
 	<thead>
 		<tr>
-			<th class="gp-column-item"><?php _ex( 'Item', 'glossary entry', 'glotpress' ); ?></th>
-			<th class="gp-column-part-of-speech"><?php _ex( 'Part of speech', 'glossary entry', 'glotpress' ); ?></th>
-			<th class="gp-column-translation"><?php _ex( 'Translation', 'glossary entry', 'glotpress' ); ?></th>
-			<th class="gp-column-comments"><?php _ex( 'Comments', 'glossary entry', 'glotpress' ); ?></th>
+			<th class="gp-column-item" data-sorter="text"><?php _ex( 'Item', 'glossary entry', 'glotpress' ); ?></th>
+			<th class="gp-column-part-of-speech" data-sorter=false><?php _ex( 'Part of speech', 'glossary entry', 'glotpress' ); ?></th>
```
Comment (ocean90) [inline gp-templates/glossary-view.php]:
```suggestion
			<th class="gp-column-part-of-speech" data-sorter="false"><?php _ex( 'Part of speech', 'glossary entry', 'glotpress' ); ?></th>
```

**Review by ocean90 — CHANGES_REQUESTED** [review]

File: assets/js/glossary.js (hunk 1 above)
Comment (pedro-mendonca) [reply assets/js/glossary.js]: Agree, doing it.

**Review by ocean90 — APPROVED** [review]

**Pull request merged**